modules/admin_manual/pages/installation/deployment_recommendations.adoc

```diff
@@ -135,55 +133,50 @@ Alternatively, you can make nightly backups — with service interruption — as
 4.  Push database dump to backup.
 5.  Start Apache.
 
-After these steps have been completed, then, optionally, rsync the backup to either an external backup storage or tape backup.
-See xref:maintenance/index.adoc[the Maintenance section] of the Administration manual for tips on backups and restores.
+After these steps have been completed, then, optionally, rsync the backup to either an external backup storage or tape backup. See xref:maintenance/index.adoc[the Maintenance section] of the Administration manual for tips on backups and restores.
 
 ==== Authentication
 
-User authentication via one or several LDAP or Active Directory (AD) servers. 
-See xref:admin_manual:configuration/user/user_auth_ldap.adoc[User Authentication with LDAP] for information on configuring ownCloud to use LDAP and AD.
+User authentication via one or several LDAP or Active Directory (AD) servers. See xref:admin_manual:configuration/user/user_auth_ldap.adoc[User Authentication with LDAP] for information on configuring ownCloud to use LDAP and AD.
 
 ==== Session Management
 
-Local session management on the application server. 
-PHP sessions are stored in a temporary filesystem, mounted at the operating system-specific session storage location. 
-You can find out where that is by running `grep -R 'session.save_path' /etc/php*` and then add it to the `/etc/fstab` file, for example:
+xref:configuration/server/caching_configuration.adoc#redis[Redis] is recommended and can be used for the session management storage.
 
-[source,console,subs="attributes+"]
-----
-include::{examplesdir}installation/deployment_recommendations/set_session_path.sh[]
-----
+Alternatively you can use local session management, see xref:local-session-management[Local Session Management].
 
 ==== Memory Caching
 
-A memory cache speeds up server performance, and ownCloud supports four of them. 
-Refer to xref:admin_manual:configuration/server/caching_configuration.adoc[Configuring Memory Caching] for information on selecting and configuring a memory cache.
+A memory cache speeds up server performance, and ownCloud supports four of them. Refer to xref:admin_manual:configuration/server/caching_configuration.adoc[Configuring Memory Caching] for information on selecting and configuring a memory cache.
```

Comment:
"four of them" - that count will go out-of-date as caching_configuration.adoc gets updated in future. Maybe:
```suggestion
A memory cache speeds up server performance, and ownCloud supports a number of them. Refer to xref:admin_manual:configuration/server/caching_configuration.adoc[Configuring Memory Caching] for information on selecting and configuring a memory cache.
```